cmd/blockchaincmd/create.go

```diff
@@ -108,6 +113,9 @@ configuration, pass the -f flag.`,
 	cmd.Flags().BoolVar(&createFlags.useExternalGasToken, "external-gas-token", false, "use a gas token from another blockchain")
 	cmd.Flags().BoolVar(&createFlags.proofOfAuthority, "proof-of-authority", false, "use proof of authority for validator management")
 	cmd.Flags().BoolVar(&createFlags.proofOfStake, "proof-of-stake", false, "use proof of stake for validator management")
+	cmd.Flags().BoolVar(&createFlags.validatorManagerMintOnly, "validator-manager-mint-only", false, "only enable validator manager contract to mint new native tokens")
```

Comment:
lets just remove this flags:
- both managers should be always minter enabled addresses in native minter precompile
- let's the user decide if adding more minters (double check with meag but It seems to me restrictive to force only one minter for PoS)